cmd/snap-exec/main_test.go

```diff
@@ -215,7 +215,7 @@ func (s *snapExecSuite) TestSnapExecAppRealIntegration(c *C) {
 	})
 
 	canaryFile := filepath.Join(c.MkDir(), "canary.txt")
-	script := filepath.Join(dirs.GlobalRootDir, "/snap/snapname/42/run-app")
+	script := fmt.Sprintf("%s/snapname/42/run-app", dirs.SnapMountDir)
```

Comment:
That's why my question: changing `dirs.GlobalRootDir` to `dirs.SnapMountDir` and removing `/snap` seems like a much quicker edit than switching to `fmt.Sprintf`